### PR TITLE
Add parameter to DQN for initial probability of epsilon greedy exploration

### LINKED ISSUE #558: [feature request]: Initializing exploration (epsilon-greedy parameter) at value other than 1
When creating a DQN, there is an option to specify `exploration_fraction` and `exploration_final_eps`, but not the initial value.

As a result, the value is in `DQN.run()` with a linear schedule beginning at 1:

```python
# Create the schedule for exploration starting from 1.
self.exploration = LinearSchedule(schedule_timesteps=int(self.exploration_fraction * total_timesteps), initial_p=1.0, final_p=self.exploration_final_eps)
```

It would be helpful to set this initial value to something other than 1 so that users could easily specify it ahead of time. Alternatively, it would be helpful to have the option to pass in the schedule (or a custom function that took the step count as input and returned epsilon). A more simple approach would be to simply put the line that defines the `self.exploration` in `__init__` rather than `run` so that a savvy user could overwrite `self.exploration`.

### PR DESCRIPTION
**Pull request by jdossgollin**
## Description

Creates a parameter of the DQN for the initial value of the random action probability. This functionality mirrors the existing parameter for the final value of the random action probability. The default initial random action probability remains 1.0, but users can now modify that value.

## Motivation and Context

closes #558 

- [x] I have raised an issue to propose this change ([required](https://github.com/hill-a/stable-baselines/blob/master/CONTRIBUTING.md) for new features and bug fixes)

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)

## Checklist:
- [x] I've read the [CONTRIBUTION](https://github.com/hill-a/stable-baselines/blob/master/CONTRIBUTING.md) guide (**required**)
- [x] I have updated the [changelog](https://github.com/hill-a/stable-baselines/blob/master/docs/misc/changelog.rst) accordingly (**required**).
- [ ] My change requires a change to the documentation.
- [x] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/misc/changelog.rst [hunk 1]
```diff
@@ -18,6 +18,7 @@ New Features:
 ^^^^^^^^^^^^^
 - Add `n_cpu_tf_sess` to model constructor to choose the number of threads used by Tensorflow
 - `VecNormalize` now supports being pickled and unpickled.
+- Add parameter `exploration_initial_eps` to DQN to mirror `exploration_final_eps`, allowing user to start training with random action probability which is not 1 (default remains 1)
```
Comment (Miffyli) [inline docs/misc/changelog.rst]: You can add your nick here in the end, and also remember to update contributors list at the end of the file.

**Comment by Miffyli** [comment]
Looks good to me :). I will wait for the tests to complete, after which I will simplify the wording on changelog and add comment on testing negative `initial_p` (a peculiar scenario, but could be useful).
